mdsshr/MdsEvents.c

```diff
@@ -199,7 +216,7 @@ static void ReconnectToServer(int idx, int recv)
     return;
   DisconnectFromMds_(ids[idx]);
   ids[idx] = ConnectToMds_(servers[idx]);
-  if (ids[idx] <= 0)
```

Comment:
maybe its worth using the constant INVALID_CONNECTION_ID  instead.  is this conform?

Reply:
True, corrected